frontend/src/App.tsx

```diff
@@ -0,0 +1,111 @@
+import { bind } from 'decko';
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import { connect, DispatchProp } from 'react-redux';
+
+// Bootstrap components
+import Container from 'react-bootstrap/Container';
+
+// CSS and JS
+import 'jquery';
+import 'bootstrap/dist/js/bootstrap.min.js';
+
+import 'font-awesome/css/font-awesome.min.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'bootswatch/dist/darkly/bootstrap.min.css';
+import './css/App.css';
+
+// Component Includes
+import Navbar from './components/fragments/global/navbar.component';
+import Footer from './components/fragments/global/footer.compontent';
+import Homepage from './components/pages/misc/homepage.component';
+import About from './components/pages/misc/about.component';
+import Profile from './components/pages/user/profile.component';
+
+import { IAppState, IUser } from 'store/reduxState';
```

Comment:
This is how you do "typed" redux stores.  In the existing code, it seems to mostly use the entire state.  Generally you only want to map the fields you need.